easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-5.8.1-intel-2021b-METIS-5.1.0.eb

```diff
@@ -0,0 +1,31 @@
+name = 'SuiteSparse'
+version = '5.8.1'
```

Comment:
We already have SuiteSparse 5.10.1 (https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-5.10.1-foss-2021b-METIS-5.1.0.eb) in the `foss/2021b` toolchain. Can we match that version here for the `intel/2021b` toolchain?

Reply:
For
```
icc: error #10106: Fatal error in /kyukon/home/apps/RHEL8/cascadelake-ib/software/intel-compilers/2021.4.0/compiler/2021.4.0/linux/bin/intel64/../../bin/intel64/mcpcom, terminated by kill signal
```
my first guess is out of memory.

I tried a test build and I also failed in the same way, but I'd got to 7% of that section done. I am now trying with more memory and fewer cores.

Reply:
So yes, with more memory the installation completes. I saw it peak at 125GB when building!

Reply:
I see, should I use `parallel = 1` in the easyconfig just to ease things out?

Reply:
For SuiteSparse that will not help. The issue is the generation of an optimisation report. See https://github.com/easybuilders/easybuild-easyconfigs/pull/17285#issuecomment-1474125285

Reply:
c1187ae